runelite-client/src/main/java/net/runelite/client/game/AgilityShortcut.java

```diff
@@ -184,8 +185,8 @@
 	FOSSIL_ISLAND_HARDWOOD_NORTH(70, "Hole" , new WorldPoint(3712, 3828, 0), HOLE_31481, HOLE_31482),
 	FOSSIL_ISLAND_HARDWOOD_SOUTH(70, "Hole" , new WorldPoint(3714, 3816, 0), HOLE_31481, HOLE_31482),
 	AL_KHARID_WINDOW(70, "Window", new WorldPoint(3293, 3158, 0), BROKEN_WALL_33344, BIG_WINDOW),
-	GWD_SARADOMIN_ROPE_NORTH(70, "Rope Descent", new WorldPoint(2912, 5300, 0), NULL_26371),
-	GWD_SARADOMIN_ROPE_SOUTH(70, "Rope Descent", new WorldPoint(2951, 5267, 0), NULL_26375),
+	GWD_SARADOMIN_ROPE_NORTH(70, "Rope Descent", new WorldPoint(2912, 5300, 2), NULL_26371, NULL_26561),
+	GWD_SARADOMIN_ROPE_SOUTH(70, "Rope Descent", new WorldPoint(2920, 5276, 1), NULL_26375, NULL_26562),
```

Comment:
The given WorldPoint is where the agility icon resides on the world map, not the location of the obstacle ingame. This change breaks the tooltip that's currently set there.